src/Generation/Generator/Renderer/Public/ReturnType/From/Utf8StringReturnType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Generator.Model;
+
+namespace Generator.Renderer.Public;
+
+internal class Utf8StringReturnType : RenderableScalarReturnType<GirModel.Utf8String>
+{
+    public override RenderableReturnType Create(GirModel.ReturnType returnType)
+    {
+        // new RenderableReturnType("string");
+        return new RenderableReturnType(Type.GetName(returnType.AnyType.AsT0));
```

Comment:
Actually you can write like mentioned in your comment and remove the comment instead:

```CS
return new RenderableReturnType("string");
```

I think this is okay as it is a lot better to understand than the current cryptic expression.

But one thing is missing. The standard parameter supported the nullable annotation for return types:

`Type.GetName(type) + Nullable.Render((GirModel.Nullable) returnType)`
I suspect that it is needed here, too. So it should probably be similar to:

```CS
return new RenderableReturnType("string" + Nullable.Render(returnType));
```